File: src/main/java/br/com/jiratorio/domain/response/UserConfigResponse.java
package br.com.jiratorio.domain.response;

import br.com.jiratorio.domain.ChartType;
import lombok.Data;

@Data
public class UserConfigResponse {

    private String state;

    private String city;

    private String holidayToken;

    private String leadTimeChartType = ChartType.BAR.name();

    private String throughputChartType = ChartType.DOUGHNUT.name();

}

File: src/main/java/br/com/jiratorio/domain/response/UserConfigResponse.kt
package br.com.jiratorio.domain.response

import br.com.jiratorio.domain.chart.ChartType

data class UserConfigResponse(
    var state: String? = null,
    var city: String? = null,
    var holidayToken: String? = null,
    var leadTimeChartType: String = ChartType.BAR.name,
    var throughputChartType: String = ChartType.DOUGHNUT.name
)
